test/passenger_test.rb

```diff
@@ -68,7 +92,33 @@
     end
   end
 
-  describe "net_expenditures" do
-    # You add tests for the net_expenditures method
+  describe "net expenditures" do
```

Comment:
For `total_time_spent` and `net_expenditures` you should also include a test case for when the passenger has in progress trips (just as you did for the driver).